src/Couchbase.Lite.Mapping.Enterprise.Tests/Couchbase.Lite.Mapping.Enterprise.Tests.csproj

```diff
@@ -17,5 +17,8 @@
   <ItemGroup>
     <ProjectReference Include="..\Couchbase.Lite.Mapping\Couchbase.Lite.Mapping.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="SimpleObjectWithArrayEnum.cs" />
+  </ItemGroup>

```

Comment:
What is the reason for this?

Reply:
No reason to be there. I added the file in the wrong place then moved it. 
I'll get rid of it.